src/ess/livedata/config/instruments/bifrost.py

```diff
@@ -226,42 +214,7 @@ def _spectrum_view_new(params: BifrostWorkflowParams) -> StreamProcessorWorkflow
     )
 
 
-@instrument.register_workflow(
-    name='counts_per_angle',
-    version=1,
-    title='Counts per angle',
-    source_names=_source_names,
-    aux_source_names=['detector_rotation'],
-)
-def _counts_per_angle() -> StreamProcessorWorkflow:
-    return StreamProcessorWorkflow(
-        _reduction_workflow.copy(),
-        dynamic_keys={'unified_detector': NeXusData[NXdetector, SampleRun]},
-        context_keys={'detector_rotation': DetectorRotation},
-        target_keys=(CountsPerAngle,),
-        accumulators=(CountsPerAngle,),
-    )
```

Comment:
It is still possible to make such views using a regular reduction workflow. However, this can now mainly be done using the secondary mechanism based on timeseries.

Reply:
It was a demo, showing that we can make correlation histograms this way. Now there is a dedicated/generic way, so this could be removed. Right?